### PR TITLE
Preparation for stable release

### PR DESCRIPTION
**Pull request by NikAzanov**
* Improve grammar and structure of documentation
* Extend docs it with examples
* Convert `README.rst` to markdown format
* Update Django & Python versions
* Update `CONTRIBUTING.rst`

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -62,31 +61,39 @@ Ready to contribute? Here's how to set up `django-import-export-extensions` for
...
+3. Setup virtual environment:
 
-    $ pyenv install 3.11
-    $ pyenv shell $(pyenv latest 3.11)
-    $ poetry config virtualenvs.in-project true
-    $ poetry env use $(which python) &&  poetry install && source .venv/bin/activate
+  Using pyenv::
+
+    pyenv install 3.12
+    pyenv shell $(pyenv latest 3.12)
```
Comment (TheSuperiorStanislav) [inline CONTRIBUTING.rst:71]:
```suggestion
    pyenv install 3.13
    pyenv shell $(pyenv latest 3.13)
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -62,31 +61,39 @@ Ready to contribute? Here's how to set up `django-import-export-extensions` for
...
+    poetry config virtualenvs.in-project true
+    source .venv/bin/activate && poetry install
+
+  Using uv::
+
+    uv venv --python 3.12 --prompt django-import-export-extensions --seed
```
Comment (TheSuperiorStanislav) [inline CONTRIBUTING.rst:77]:
```suggestion
    uv venv --python 3.13 --prompt django-import-export-extensions --seed
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+Celery](https://docs.celeryq.dev/en/stable/django/first-steps-with-django.html)
+if you want to run import/export in background
+
+## Links
+
+- Documentation:
+    <https://django-import-export-extensions.readthedocs.io>.
+- GitHub:
+    <https://github.com/saritasa-nest/django-import-export-extensions/>
+- PyPI: <https://pypi.org/project/django-import-export-extensions/>
```
Comment (TheSuperiorStanislav) [inline README.md:114]: Let's move this up, I think people would to see links to docs first, also add link to history and contributing maybe?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+|-----------|:----|--------|
+| Versions  |     | [![PyPI - Python Versions](https://img.shields.io/pypi/pyversions/django-import-export-extensions)](https://pypi.org/project/django-import-export-extensions/)<br>[![PyPI - Django Versions](https://img.shields.io/pypi/frameworkversions/django/django-import-export-extensions)](https://pypi.org/project/django-import-export-extensions/) |
+| CI/CD     |     | [![Build status on Github](https://github.com/saritasa-nest/django-import-export-extensions/actions/workflows/checks.yml/badge.svg)](https://github.com/saritasa-nest/django-import-export-extensions/actions/workflows/checks.yml)<br>[![Test coverage](https://coveralls.io/repos/github/saritasa-nest/django-import-export-extensions/badge.svg?branch=main)](https://coveralls.io/github/saritasa-nest/django-import-export-extensions?branch=main)<br>[![Documentation Status](https://readthedocs.org/projects/django-import-export-extensions/badge/?version=latest)](https://django-import-export-extensions.readthedocs.io/en/latest/?badge=latest) |
+| Statistic |     | ![PyPI Downloads](https://static.pepy.tech/badge/django-import-export-extensions/month) |
+
+## Description
```
Comment (TheSuperiorStanislav) [inline README.md:9]: What do you thinkl about adding table of contents https://github.com/saritasa-nest/saritasa-invocations#table-of-contents

File: README.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+class BookExportViewSet(views.ExportJobViewSet):
+    resource_class = resources.BookResource
+
+
+class BookImportViewSet(views.ImportJobViewSet):
+    resource_class = resources.BookResource
+```
+
+Don\'t forget to [configure
```
Comment (Eg0ra) [inline README.md:104]:
```suggestion
Don't forget to [configure
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+| Project   |     | Status |
+|-----------|:----|--------|
+| Versions  |     | [![PyPI - Python Versions](https://img.shields.io/pypi/pyversions/django-import-export-extensions)](https://pypi.org/project/django-import-export-extensions/)<br>[![PyPI - Django Versions](https://img.shields.io/pypi/frameworkversions/django/django-import-export-extensions)](https://pypi.org/project/django-import-export-extensions/) |
+| CI/CD     |     | [![Build status on Github](https://github.com/saritasa-nest/django-import-export-extensions/actions/workflows/checks.yml/badge.svg)](https://github.com/saritasa-nest/django-import-export-extensions/actions/workflows/checks.yml)<br>[![Test coverage](https://coveralls.io/repos/github/saritasa-nest/django-import-export-extensions/badge.svg?branch=main)](https://coveralls.io/github/saritasa-nest/django-import-export-extensions?branch=main)<br>[![Documentation Status](https://readthedocs.org/projects/django-import-export-extensions/badge/?version=latest)](https://django-import-export-extensions.readthedocs.io/en/latest/?badge=latest) |
+| Statistic |     | ![PyPI Downloads](https://static.pepy.tech/badge/django-import-export-extensions/month) |
```
Comment (TheSuperiorStanislav) [inline README.md:7]: In invocations it looks more readable, also let's add extra one(from saritasa invocations)

GitHub Workflow Status (with event)
PyPI
PyPI - Status
PyPI - Python Version
PyPI - Django Version
PyPI - License
PyPI - Downloads
[Ruff](https://github.com/astral-sh/ruff)

File: README.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+adding the following features:
+
+- Import/export resources in the background via Celery
+- Manage import/export jobs via Django Admin
+- DRF integration that allows to work with import/export jobs via API
+- Support
+    [drf-spectacular](https://github.com/tfranzel/drf-spectacular)
+    generated API schema
+- Additional fields and widgets (FileWidget,
+    IntermediateManyToManyWidget, IntermediateManyToManyField)
```
Comment (Eg0ra) [inline README.md:22]:
```suggestion
- Support [drf-spectacular](https://github.com/tfranzel/drf-spectacular)
generated API schema
- Additional fields and widgets (FileWidget, IntermediateManyToManyWidget,
IntermediateManyToManyField)
```

File: README.md (hunk 3 above)
Comment (Eg0ra) [inline README.md:114]:
```suggestion
- Documentation: <https://django-import-export-extensions.readthedocs.io>
- GitHub: <https://github.com/saritasa-nest/django-import-export-extensions/>
- PyPI: <https://pypi.org/project/django-import-export-extensions/>
```

File: pyproject.toml [hunk 8]
```diff
@@ -26,18 +26,25 @@ keywords = [
 classifiers = [
   "Development Status :: 4 - Beta",
```
Comment (TheSuperiorStanislav) [inline pyproject.toml:27]:
```suggestion
  "Development Status :: 5 - Production/Stable",,
```

File: pyproject.toml [hunk 9]
```diff
@@ -26,18 +26,25 @@ keywords = [
...
   "Framework :: Django :: 5.0",
+  "Framework :: Django :: 5.1",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: 3 :: Only",
```
Comment (TheSuperiorStanislav) [inline pyproject.toml:41]: No need for this, poetry sets it automatically, just when publishing make sure you are on python 3.13

File: pyproject.toml [hunk 10]
```diff
@@ -26,18 +26,25 @@ keywords = [
...
   "Development Status :: 4 - Beta",
   "Intended Audience :: Developers",
-  "Natural Language :: English",
+  "Topic :: Software Development",
   "Framework :: Django",
-  "Framework :: Django :: 4.0",
-  "Framework :: Django :: 4.1",
   "Framework :: Django :: 4.2",
   "Framework :: Django :: 5.0",
+  "Framework :: Django :: 5.1",
```
Comment (TheSuperiorStanislav) [inline pyproject.toml:33]: image
```suggestion
  "Framework :: Django :: 5.1",
  "Framework :: Django :: 5.2",
```

File: pyproject.toml [hunk 11]
```diff
@@ -26,18 +26,25 @@ keywords = [
...
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: 3 :: Only",
 ]
 packages = [{ include = "import_export_extensions" }]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-django = ">=4.0"
+django = ">=4.2"
```
Comment (TheSuperiorStanislav) [inline pyproject.toml:47]: Need to reflect it in changelog I guess

File: docs/getting_started.rst [hunk 12]
```diff
@@ -2,20 +2,20 @@
 Getting started
 ===============
 
-``django-import-export-extensions`` is based on ``django-import-export`` package so it have similar
-workflow and interfaces. If you already worked with original one,
-you can check :ref:`Migrate from original django-import-export package<Migrate from original \`django-import-export\` package>`
+``django-import-export-extensions`` is based on ``django-import-export`` package,
+so it follows a similar workflow and interfaces.f you are already familiar with the original package,
```
Comment (OttoAndrey) [inline docs/getting_started.rst:6]:
```suggestion
so it follows a similar workflow and interfaces. If you are already familiar with the original package,
```

File: README.md (hunk 6 above)
Comment (NikAzanov) [reply README.md:7]: Code - yes
Format - I believe that table is more convenient

image

File: README.md (hunk 4 above)
Comment (NikAzanov) [reply README.md:9]: [The documentation](https://django-import-export-extensions.readthedocs.io/en/latest/index.html) already contains a table of contents, so I think it's better to keep `README.md` small

File: README.md (hunk 6 above)
Comment (TheSuperiorStanislav) [reply README.md:7]: It's just usually all others do one liner
image
image
image

File: docs/installation.rst [hunk 13]
```diff
@@ -8,33 +8,44 @@ Installation and configuration
...
-you through the process.
+    $ uv pip install django-import-export-extensions
+
+Using poetry:
+
+.. code-block:: shell
 
-.. _pip: https://pip.pypa.io
-.. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
+    $ poetry add django-import-export-extensions
```
Comment (OttoAndrey) [inline docs/installation.rst:29]: If you deleted `$` sign in `contributing.rst` so maybe you should delete it here too?
```suggestion
    poetry add django-import-export-extensions
```

File: .github/workflows/checks.yml [hunk 14]
```diff
@@ -13,9 +13,8 @@ jobs:
           - "3.10"
           - "3.11"
           - "3.12"
+          - "3.13"
         django-version:
-          - ">=4.0,<4.1"
-          - ">=4.1,<4.2"
           - ">=4.2,<4.3"
           - ">=5.0,<5.2"
```
Comment (TheSuperiorStanislav) [inline .github/workflows/checks.yml:19]: Let's add Django 5.2 here

File: pyproject.toml (hunk 10 above)
Comment (TheSuperiorStanislav) [reply pyproject.toml:33]: Opps, my bad django 5.2 only expected in April 2025

File: README.md (hunk 6 above)
Comment (NikAzanov) [reply README.md]: Personally I don't like one liners with lots of items

I suggest making it this way:

image


Simple in code, looks better in UI

**Review by OttoAndrey — APPROVED** [review]

File: pyproject.toml (hunk 9 above)
Comment (NikAzanov) [reply pyproject.toml:41]: What versions will be set by poetry?

I didn't find anything about it in poetry docs. So I'm not sure if we should make it implicit.

Comment (TheSuperiorStanislav) [reply pyproject.toml:41]: From min allowed to the one you use as far as I understand
You can unzip file located in dist with `.whl` extension then 
image
image

**Review by TheSuperiorStanislav — APPROVED** [review]

**Pull request merged**